akeneo-design-system/src/components/Checkbox/Checkbox.tsx

```diff
@@ -1,7 +1,21 @@
-import React from 'react';
+import React, {Ref, useState} from 'react';
 import styled, {css, keyframes} from 'styled-components';
 import {AkeneoThemedProps, getColor} from 'theme';
 import {CheckIcon, PartialCheckIcon} from 'icons';
+import {useShortcut} from 'hooks/use-shortcut';
+import {Key} from 'shared/key';
```

Comment:
```suggestion
import {Key, uuid} from 'shared';
```
Same, with an `index.ts` in shared folder